src/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -326,7 +326,7 @@ private NewGameChooserEntry selectByName(final Component ui, final boolean force
     final String userPreferredDefaultGameName =
         (forceFactoryDefault ? DEFAULT_GAME_NAME : prefs.get(DEFAULT_GAME_NAME_PREF, DEFAULT_GAME_NAME));
     final NewGameChooserModel model = NewGameChooser.getNewGameChooserModel();
-    selectedGame = model.findByName(userPreferredDefaultGameName);
+//    selectedGame = model.findByName(userPreferredDefaultGameName);
```

Comment:
No, was testing out the default map selection logic. Which now assumes a map that does not exist, but it looks like the handling/logic for it is broken anyways. This debug change is now removed .